sdk/containerregistry/Microsoft.Azure.Management.ContainerRegistry/src/Generated/SdkInfo_ContainerRegistryManagementClient.cs

```diff
@@ -19,13 +19,19 @@ internal static partial class SdkInfo
           {
               return new Tuple<string, string, string>[]
               {
-                new Tuple<string, string, string>("ContainerRegistry", "Operations", "2019-05-01"),
-                new Tuple<string, string, string>("ContainerRegistry", "Registries", "2019-05-01"),
+                new Tuple<string, string, string>("ContainerRegistry", "AgentPools", "2019-06-01-preview"),
+                new Tuple<string, string, string>("ContainerRegistry", "Operations", "2019-12-01-preview"),
```

Comment:
Since you are adding new api versions, please update the api tags in `AzSdk.RP.props`